sqlalchemy_utils/functions/database.py

```diff
@@ -420,10 +421,12 @@ def is_auto_assigned_date_column(column):
     )
 
 
-def database_exists(url):
+def database_exists(url, postgres_db=None):
     """Check if a database exists.
 
     :param url: A SQLAlchemy engine URL.
+    :postgres_db: Only applies to postgres. List of databases to try to connect
```

Comment:
Do this parameter apply to any other databases? In other words does for example Oracle or MSSQL have similar things that a default database should be provided in order to check the existence of another database. If so, we should use some different naming for this.

Furthermore the description of the parameter suggests this parameter is a list of values but the name does not. Either the parameter should be named ``postgres_databases`` (or something similar) OR this parameter could be string. 

I'm leaning towards the latter option as the end user who is using this function should know which default database to use and just provide that as a string parameter for this function.

Reply:
> Do this parameter apply to any other databases?

I have no idea. But it seems to be postgres specific (I also found no issue that indicates such a problem for other databases). Still - if desired - I could change the parameter name  to `databases` and indicate in the docs that it currently only applies to postgres.

> Furthermore the description of the parameter suggests this parameter is a list of values but the name does not.

I would go for `postgres_databases`. The list has the advantage will work for many settings automatically.